src/match.js

```diff
@@ -171,7 +171,12 @@ function findAttributesPattern (priority, element, ignore) {
       if (element.classList.length === 0) {
         continue;
       }
-      const className = attributeValue.trim().replace(/\s+/g, '.')
+      const className = attributeValue.trim().split(/\s+/).filter((val) => {
+        return /\D/.test(val[0]);
+      }).join('.');  
+          
+      if (!className) continue;
```

Comment:
I think there can be a little bit more clarity around what's happening — the `!className` inherently means that there were no valid class names, but it's not totally clear from the code that's what you mean. 

I wonder if we do something like:

```js
const validClassNames = attributeValue.trim().split(/\s+/).filter((val) => {
  return /\D/.test(val[0]);
});

if (validClassNames.length === 0) continue;

pattern = `.${validClassNames.join('.')}`
```